src/cljs/ataru/hakija/application_hakukohde_component.cljs

```diff
@@ -79,7 +81,7 @@
 
 (defn- selected-hakukohde-row
   [hakukohde-oid]
-  [:div.application__hakukohde-row.application__hakukohde-row--selected
+  [:div.application__hakukohde-row.application__hakukohde-row--selected.animated.fadeIn
```

Comment:
Täällä vois sitten jollain `hakukohde-animation` subscriptionilla lukea että onko fade in vai out.

Reply:
Puhetta oli ettei viitsi tehdä lisää metatietoa sinne db:een, mutta onhan toi aika näästi että suoraan interopilla laittaa classnameja paikalleen.